### ISSUE TITLE
Trainable BERT +  Small GCN [ROUND 3]

### ISSUE THREAD

**Issue by balthazarneveu** [balthazarneveu/molecule-retrieval-using-nlp#14]
- Start from scratch (no pretrained model)

>  Maximum batch size on current
> - KAGGLE : 64 (OOM at 96 during validation)
> - RTX2060 : 32 

- [ ] Try dropout (in GCN?)
- [x] Larger `batch_size` : go smoothly ... CLIP used batch_size = 32k ... there's probably a workaround here to improve batch_size
- [x] Implement mixed precision or half precision.
- [ ] Use : https://github.com/huggingface/accelerate ?
- from ROUND 2, the idea of using LORA also seemed like a good idea

:cool:  Be patient : Try 120 epochs
- [x] Implement code to resume training (including optimizer state)
- [x] short code to resume experiment

:warning:  Ok locally, ok on collab.
:x:  more difficult to resume under Kaggle (probably update a model through kaggle API)
